src/modules/components/SearchForm.tsx

```diff
@@ -21,7 +21,7 @@ import styles from "@patternfly/react-styles/css/components/Form/form";
 export interface FormProps {
 	defaultValues?: FormInputs;
 	isLoading: boolean;
-	onSubmit: (query: FormInputs) => void;
+	onSubmit: (_query: FormInputs) => void;
```

Comment:
Apologies, as the lint warnings for this file and the following were driving me nuts so I made the fixes, but this change and the next are irrelevant to the PR